pybamm/__init__.py

```diff
@@ -214,38 +160,38 @@
 from .solvers.idaklu_jax import IDAKLUJax
 from .solvers.idaklu_solver import IDAKLUSolver, have_idaklu
 
-#
-# Experiments
-#
 from .experiment.experiment import Experiment
 from . import experiment
 from .experiment import step
 
-
-#
-# Plotting
-#
 from .plotting.quick_plot import QuickPlot, close_plots, QuickPlotAxes
 from .plotting.plot import plot
 from .plotting.plot2D import plot2D
 from .plotting.plot_voltage_components import plot_voltage_components
 from .plotting.plot_summary_variables import plot_summary_variables
 from .plotting.dynamic_plot import dynamic_plot
 
-#
-# Simulation
-#
 from .simulation import Simulation, load_sim, is_notebook
 
-#
-# Batch Study
-#
 from .batch_study import BatchStudy
 
+from . import callbacks
+
 #
-# Callbacks
+# Constants
 #
-from . import callbacks
+# Float format: a float can be converted to a 17 digit decimal and back without
+# loss of information
+FLOAT_FORMAT = "{: .17e}"
+# Absolute path to the PyBaMM repo
+script_path = os.path.abspath(__file__)
+
+ABSOLUTE_PATH = root_dir()
+PARAMETER_PATH = [
+    root_dir(),
+    os.getcwd(),
+    os.path.join(root_dir(), "pybamm", "input", "parameters"),
+]
```

Comment:
@tinosulzer These don't appear to be used anywhere are they OK to remove as well?